### PR TITLE
Reduce padding around plugin list titles for marketing tools page.

### LINKED ISSUE #8164: Too much padding in `Set up marketing tools` task
### Describe the bug
Super minor UI bug, I think `REACH OUT TO CUSTOMERS` has too much padding

image

### To Reproduce

1. Go to WooCommerce > Home
2. In task list, click on Set up marketing tools
3. Observe the thing

### PR DESCRIPTION
**Pull request by jacob-sewell**
Fixes #8164 

Reduce padding around plugin list titles for marketing tools page. (In the screenshots below, we're talking about "Reach out to customers" and "Grow your store".)

### Screenshots

Before:
Screen Shot 2022-01-14 at 12 23 37

After:
Screen Shot 2022-01-14 at 12 23 57

### Detailed test instructions:

1. Go to WooCommerce > Home
2. In task list, click on Set up marketing tools
3. Observe the padding around "Reach out to customers" and "Grow your store".

### CODE REVIEW

**Comment by jacob-sewell** [comment]
Small padding change. I'm particularly interested in from the creator of the issue, @ilyasfoo.

File: client/tasks/fills/Marketing/PluginList.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .woocommerce-plugin-list__title {
-	padding: $gap-larger 30px $gap 30px;
+	padding: $gap 30px 0 30px;
```
Comment (ilyasfoo) [inline client/tasks/fills/Marketing/PluginList.scss:2]:
```suggestion
	padding: $gap-large 30px 0 30px;
```

**Review by ilyasfoo — APPROVED** [review]
Thanks, @jacob-sewell! I've looked at it and I think `gap-large` for top looks a tad better.

image

Pre-approving! 🍏

**Comment by mattsherman** [comment]
@jacob-sewell Do we need another review on this, or is it ready to be merged?

**Comment by jacob-sewell** [comment]
> @jacob-sewell Do we need another review on this, or is it ready to be merged?

No, it's ready, I just hadn't noticed I finally had enough passing tests.

**Pull request merged**